components/CallToAction/CallToAction.styles.tsx

```diff
@@ -51,6 +51,15 @@ export const StyledCallToAction = styled.section`
             .subText {
                 font-size: 1.25rem;
             }
+            @media only screen and (max-width: 750px) {
+                h1 {
```

Comment:
Move the contents of this media query into the one below. 
```
@media {
 width: 100%;

 .subtext > h1{
  css goes here
 }
}
```

Reply:
doesnt work if i do this

Reply:
Does it not, or are you doing it wrong?

Reply:
Want to go over it? Cuz i'd be willing to bet it does work

Reply:
i thought you meant put the h1 tag inside the .subText